### PR TITLE
Fulltext search

### PR DESCRIPTION
**Pull request by maggarwal13**

### CODE REVIEW

File: whois-api/src/main/java/net/ripe/db/whois/api/fulltextsearch/ElasticFulltextSearch.java [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+                        .field("*");
+
+                final SearchSourceBuilder sourceBuilder = new SearchSourceBuilder()
+                        .query(getQueryBuilder(searchRequest.getQuery()))
+                        .size(searchRequest.getRows()).from(start)
+                        .aggregation(AGGREGATION_BUILDER)
+                        .sort(SORT_BUILDERS)
+                        .highlighter(highlightBuilder);
+
+                final org.elasticsearch.action.search.SearchRequest fulltextRequest = new org.elasticsearch.action.search.SearchRequest(elasticIndexService.getWHOIS_INDEX());
```
Comment (eshryane) [inline whois-api/src/main/java/net/ripe/db/whois/api/fulltextsearch/ElasticFulltextSearch.java:106]: Remove package from class (there's only one SearchRequest class?)

Comment (maggarwal13) [reply whois-api/src/main/java/net/ripe/db/whois/api/fulltextsearch/ElasticFulltextSearch.java:106]: It als exists in package : net.ripe.db.whois.api.fulltextsearch

Comment (eshryane) [reply whois-api/src/main/java/net/ripe/db/whois/api/fulltextsearch/ElasticFulltextSearch.java:106]: OK, if there's a duplicate class you can leave the package path

File: whois-api/src/test/java/net/ripe/db/whois/api/elasticsearch/ElasticAutocompleteServiceTestIntegration.java [hunk 2]
```diff
@@ -67,12 +67,12 @@ public void setUp() throws Exception {
     }
 
     @Override
-    String getWhoisIndex() {
+    public String getWhoisIndex() {
```
Comment (eshryane) [inline whois-api/src/test/java/net/ripe/db/whois/api/elasticsearch/ElasticAutocompleteServiceTestIntegration.java:70]: Do we need the `public` modifier?

Comment (maggarwal13) [reply whois-api/src/test/java/net/ripe/db/whois/api/elasticsearch/ElasticAutocompleteServiceTestIntegration.java:70]: Yes autocomplete and fulltext integration tests are in separate packages and both needs to be override this method

**Pull request merged**